### PR TITLE
feat:  APP-3100 - Remove DataList padding

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description

- update DataList padding to allow component to use full container width

Task: [APP-3100](https://aragonassociation.atlassian.net/browse/APP-3100)

## Type of change

-   [x] New feature (non-breaking change which adds functionality)

## Checklist:

-   [x] I have selected the correct base branch.
-   [x] I have performed a self-review of my own code.
-   [x] I have commented my code, particularly in hard-to-understand areas.
-   [x] I have made corresponding changes to the documentation.
-   [x] My changes generate no new warnings.
-   [x] Any dependent changes have been merged and published in downstream modules.
-   [x] I ran all tests with success and extended them if necessary.
-   [x] I have updated the `CHANGELOG.md` file in the root folder of the package after the [UPCOMING] title and before
        the latest version.
-   [ ] I have tested my code on the test network.

[APP-3100]: https://aragonassociation.atlassian.net/browse/APP-3100?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by cgero-eth — COMMENTED** [review]
Thanks for the update @Fabricevladimir, remember to update the changelog file 🙏🏻

File: src/core/components/dataList/dataListContainer/dataListContainer.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const DataListContainer: React.FC<IDataListContainerProps> = (props) => {
 
     return (
         <div
-            className={classNames('flex flex-col gap-2 px-0 md:px-6', className)}
+            className={classNames('flex flex-col gap-2', className)}
```
[the remark this answers is not in the document]
Comment (Fabricevladimir) [reply src/core/components/dataList/dataListContainer/dataListContainer.tsx:61]: No padding for those either!

[the remark this answers is not in the document]
Comment (cgero-eth) [reply src/core/components/dataList/dataListContainer/dataListContainer.tsx:61]: 🔥

**Review by cgero-eth — APPROVED** [review]
LGTM, thanks @Fabricevladimir 👌🏻
Minor note, looks like we're mixing this currently, but could you update the changelog entry to use present tense? 🙏🏻

**Pull request merged**